apps/rehabstod/src/components/Table/tableHeader/TableHeaderCell.tsx

```diff
@@ -4,15 +4,25 @@ import { TooltipTrigger } from '../../Tooltip/TooltipTrigger'
 import { useTableContext } from '../hooks/useTableContext'
 import { SortingIcon } from './SortingIcon'
 
-export function TableHeaderCell({ column, width, sticky }: { column: string; width?: string; sticky?: 'left' | 'top' | 'right' }) {
+export function TableHeaderCell({
+  column,
+  width,
+  sticky,
+  unSortable,
+}: {
+  column: string
+  width?: string
+  sticky?: 'left' | 'top' | 'right'
+  unSortable?: boolean
```

Comment:
I think changing this to "sortable" would be easier to reason about. It could also have a default value:

```typescript
{sortable = true}: {sortable?: boolean}
```
```typescript
tabIndex={sortable ? 0 : undefined}
```